jaffle_shop_online/subscribers.yml

```
@@ -0,0 +1,3 @@
+subscribers:
+  - asset: model.jaffle_shop_online.cpa_and_roas
+    email: user@example.com
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)